src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="searchBar"
+      action="/action_page.php"
+      method="get"
+      data-qa="big"
+    >
+      <label>
+        <img
+          class="imgBig"
+          src="./images/Search.svg"
+          alt="searchImage"
+        >
+        <input
+          class="inputBig"
+          name="searchBarLarge"
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
+    <form
+      class="searchBar"
+      data-qa="small"
+      action="/action_page.php"
+      method="get"
+    >
+      <label>
+        <img
+          class="imgSmall"
+          src="./images/Search.svg"
+          alt="searchImage"
+        >
```

Comment:
You can do this with pseudo elements, and without img tag. But it's okay

Reply:
Understand, but leave it like it was for the first time. Thank you for comment.